### DISCUSSION TITLE
疑似新版动态首页新功能

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3287] [综合讨论 / General]
![图片](https://user-images.githubusercontent.com/81268244/166125625-d8804b12-d3b2-4f1f-abb6-87c502928ae5.png)
如图，如果可以加入一个自动展开的功能就更好了
 
 

<del>（离V1脚本完全不能使用的日子又近了一些呢）</del>

**Comment by the1812** [comment]
#3191 
~~v2 下次更新就会支持~~
